README.md

```diff
@@ -117,3 +117,27 @@ or `cognoma-nginx:X` which can be used to determine which are the correct
 tasks. Once you have selected the correct tasks click the **Stop** button.
 This will cause the tasks to be stopped and ECS will restart them with the
 new version of the container you have pushed. Therefore you're now done.
+
+### Updating the Data
+
+Take a look at `api/management/commands/acquiredata.py`.
+In there are two commit hashes, one for most of the data called `COMMIT_HASH` and one just for genes data called `GENES_COMMIT_HASH`.
+Update these hashes to the hash of the data that you want to update to.
+Once you've done so you should redploy the core-service.
+Once the core-service has been redeployed you should ssh onto the EC2 instance next you should run `docker ps` to get a list of containers running on that instance.
+Find the name for the core-service container and run `docker exec -it <that_name> /bin/bash`.
+Once you're in the container run `rm data/*` to make sure that the old data is cleared out.
+Next, empty the tables you're trying to update by first running `python3 manage.py shell` and then within that python shell:
+```
```

Comment:
Would it be good to just run this as the first step to either acquiredata or loaddata management command (whichever one is more appropriate)?

Reply:
Good call!